app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @review do |f| %>
```

Comment:
<%= form_for [@project, @review] do |f| %>
...

So that we can remove that hidden_field value.

http://stackoverflow.com/questions/24173836/nested-form-with-nested-resource-rails-4